syntax.php

```diff
@@ -184,7 +184,7 @@ function render($format, Doku_Renderer $renderer, $data) {
         $data += $this->query($data['id']);
 
         // link title
-        $link = 'issue #'. $data['id'];
+        $link = $this->getLang('issue') . ' #'. $data['id'];
```

Comment:
Is create fix commit OK now? Or should I create new commits and new pull request?